### PR TITLE
Add `4 Digit World Cup 2024` article

### PR DESCRIPTION
**Pull request by Walavouchey**
[web preview](https://osu1.roanh.dev/wiki/en/Tournaments/4WC/2024)

### CODE REVIEW

File: wiki/Tournaments/4WC/2024/en.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+| Finals | 2024-06-08/2024-06-09 |
+| Grand Finals | 2024-06-15/2024-06-16 |
+
+## Prizes
+
+The 4 Digit World Cup 2024 awards winners with a [community-funded prize pool on Open Collective](https://opencollective.com/4wc).
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | Wooting UwU (non-RGB) keyboard for each player, Bluebunny tablet cover, 40% of the prize pool, profile badge (pending), animated profile banner |
```
Comment (shdewz) [inline wiki/Tournaments/4WC/2024/en.md:34]: not sure if it sounds too clunky adding "for each player" on the tablet covers as well, but should probably not just have it on one of these prizes

File: wiki/Tournaments/4WC/2024/en.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
...su.ppy.sh/users/8215374), ::{ flag=DE }:: [-semi](https://osu.ppy.sh/users/5154946), ::{ flag=PL }:: [cezgru](https://osu.ppy.sh/users/2629617), ::{ flag=US }:: [ChillierPear](https://osu.ppy.sh/users/9501251), ::{ flag=RU }:: [Endura](https://osu.ppy.sh/users/7774197), ::{ flag=PH }:: [enri](https://osu.ppy.sh/users/8640970), ::{ flag=CN }:: [FcEazy](https://osu.ppy.sh/users/7825227), ::{ flag=US }:: [Flameztear](https://osu.ppy.sh/users/13207763), ::{ flag=US }:: [Jakson](https://osu.ppy.sh/users/8788058), ::{ flag=PL }:: [kiir0chii](https://osu.ppy.sh/users/9322480), ::{ flag=US }:: [M I L E S](https://osu.ppy.sh/users/6036351), ::{ flag=GB }:: [Malteser](https://osu.ppy.sh/users/5218178), ::{ flag=ES }:: [MeiFTW](https://osu.ppy.sh/users/15049805), ::{ flag=RO }:: [nanoya](https://osu.ppy.sh/users/12366071), ::{ flag=FI }:: [Nev-](https://osu.ppy.sh/users/11836334), ::{ flag=NO }:: [ninerik](https://osu.ppy.sh/users/10549880), ::{ flag=MN }:: [seegii](https://osu.ppy.sh/users/4659319), ::{ flag=FI }:: [shdewz](https://osu.ppy.sh/users/10000899), ::{ flag=KR }:: [worst hr player](https://osu.ppy.sh/users/14106450), ::{ flag=DE }:: [ZeitFrost](https://osu.ppy.sh/users/11234706) |
```
Comment (shdewz) [inline wiki/Tournaments/4WC/2024/en.md:47]:
```suggestion
| Mappool playtester | ::{ flag=RU }:: [_wonderer](https://osu.ppy.sh/users/8215374), ::{ flag=DE }:: [-semi](https://osu.ppy.sh/users/5154946), ::{ flag=PL }:: [cezgru](https://osu.ppy.sh/users/2629617), ::{ flag=US }:: [ChillierPear](https://osu.ppy.sh/users/9501251), ::{ flag=RU }:: [Endura](https://osu.ppy.sh/users/7774197), ::{ flag=PH }:: [enri](https://osu.ppy.sh/users/8640970), ::{ flag=CN }:: [FcEazy](https://osu.ppy.sh/users/7825227), ::{ flag=US }:: [Flameztear](https://osu.ppy.sh/users/13207763), ::{ flag=US }:: [Jakson](https://osu.ppy.sh/users/8788058), ::{ flag=PL }:: [kiir0chii](https://osu.ppy.sh/users/9322480), ::{ flag=US }:: [M I L E S](https://osu.ppy.sh/users/6036351), ::{ flag=GB }:: [Malteser](https://osu.ppy.sh/users/5218178), ::{ flag=ES }:: [MeiFTW](https://osu.ppy.sh/users/15049805), ::{ flag=RO }:: [nanoya](https://osu.ppy.sh/users/12366071), ::{ flag=FI }:: [Nev-](https://osu.ppy.sh/users/11836334), ::{ flag=NO }:: [ninerik](https://osu.ppy.sh/users/10549880), ::{ flag=MN }:: [seegii](https://osu.ppy.sh/users/4659319), ::{ flag=FI }:: [shdewz](https://osu.ppy.sh/users/10000899), ::{ flag=KR }:: [worst hr player](https://osu.ppy.sh/users/14106450), ::{ flag=DE }:: [ZeitFrost](https://osu.ppy.sh/users/11234706) |
```

non-plural role name

File: wiki/Tournaments/4WC/2024/en.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+
+## Links
+
+- **[Registration form](https://osu.ppy.sh/oauth/authorize?client_id=21966&redirect_uri=https://script.google.com/macros/s/AKfycbzknySoKogThpPyGuOGOsmsr1hdCITHOQqF3UXQc-orw-9b_BuEjM129uVUBHmNGWBMpg/exec&response_type=code&scope=identify&state=%7B%22step%22%3A%22osu%22%7D)**
+- [Discussion thread](https://osu.ppy.sh/community/forums/topics/1898593)
+- [Livestream](https://twitch.tv/4wcb)
+- [Statistics sheet](https://docs.google.com/spreadsheets/d/1gPRSI1lTW8ZMlu_1mnxpVcZyzO2Atbb_2H5nfDt0CXM)
```
Comment (shdewz) [inline wiki/Tournaments/4WC/2024/en.md:57]:
```suggestion
- [Information spreadsheet](https://docs.google.com/spreadsheets/d/1gPRSI1lTW8ZMlu_1mnxpVcZyzO2Atbb_2H5nfDt0CXM)
```

more fitting in my opinion, and same convention as official world cups over the past year or so

File: wiki/Tournaments/4WC/2024/en.md [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+- [Challonge bracket](https://challonge.com/4wc24)
+- [Discord server](https://discord.com/invite/uQarcRtB3u)
+
+## Ruleset
+
+### General
+
+- This is a **country-based**, **world-cup-style** tournament.
+- This is a **4v4**, **osu!** tournament with **teams of 5—8 players**.
```
Comment (shdewz) [inline wiki/Tournaments/4WC/2024/en.md:66]:
```suggestion
- This is a **4v4**, **osu!** tournament with **teams of 5–8 players**.
```

em -> en

File: wiki/Tournaments/4WC/2024/en.md [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+- After registrations close, players are allowed to leave the rank range.
+- Match schedules and other events will be displayed in **UTC time**.
+- Matches can be rescheduled if both teams agree to a time and communicate it to the tournament organisers **before Thursday 23:59 UTC** of the week the match is to take place on.
+- Staff members, with the exception of streamers and commentators, are not allowed to play in the tournament.
+- The tournament organisers reserve the right to modify the rules at any moment.
+  - Any changes to the rules will be clearly communicated to the players.
+
+### Conduct
+
+- All spaces affiliated with 4WC are subject to the [osu! community rules](/wiki/Rules), as well as the rules of the platform itself.
```
Comment (shdewz) [inline wiki/Tournaments/4WC/2024/en.md:84]:
```suggestion
- All spaces affiliated with 4WC are subject to the [osu! community rules](/wiki/Rules), as well as the rules of the respective platforms themselves.
```

maybe this wording as discussed

File: wiki/Tournaments/4WC/2024/en.md [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+### Match format
+
+| Stage | Format |
+| :-- | :-- |
+| Round of 32 | Best of 9, 1 ban |
+| Round of 16 | Best of 9, 1 ban |
+| Quarterfinals | Best of 11, 2 bans |
+| Semifinals | Best of 11, 2 bans |
+| Finals | Best of 13, 2 bans |
+| Grand Finals | Best of 132, bans |
```
Comment (shdewz) [inline wiki/Tournaments/4WC/2024/en.md:111]:
```suggestion
| Grand Finals | Best of 13, 2 bans |
```

File: wiki/Tournaments/4WC/2024/en.md (hunk 1 above)
Comment (Walavouchey) [reply wiki/Tournaments/4WC/2024/en.md:34]: just forgot for some reason

```suggestion
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | Wooting UwU (non-RGB) keyboard and Bluebunny tablet cover for each player, 40% of the prize pool, profile badge (pending), animated profile banner |
```

File: wiki/Tournaments/4WC/2024/en.md [hunk 7]
```diff
@@ -0,0 +1,150 @@
...
+| Streamer | ::{ flag=US }:: [D I O](https://osu.ppy.sh/users/3958619), ::{ flag=VN }:: [Hoaq](https://osu.ppy.sh/users/7696512), ::{ flag=GB }:: [ilw8](https://osu.ppy.sh/users/14167692), ::{ flag=CA }:: [kymotsujason](https://osu.ppy.sh/users/2541804), ::{ flag=US }:: [ROB_](https://osu.ppy.sh/users/12455868), ::{ flag=FI }:: [shdewz](https://osu.ppy.sh/users/10000899), ::{ flag=VN }:: [SIay](https://osu.ppy.sh/users/9587896), ::{ flag=US }:: [this1neguy](https://osu.ppy.sh/users/1797189) |
```
Comment (iSlodinxOsu) [inline wiki/Tournaments/4WC/2024/en.md:48]:
```suggestion
| Streamer | ::{ flag=CA }:: [D I O](https://osu.ppy.sh/users/3958619), ::{ flag=VN }:: [Hoaq](https://osu.ppy.sh/users/7696512), ::{ flag=GB }:: [ilw8](https://osu.ppy.sh/users/14167692), ::{ flag=CA }:: [kymotsujason](https://osu.ppy.sh/users/2541804), ::{ flag=US }:: [ROB_](https://osu.ppy.sh/users/12455868), ::{ flag=FI }:: [shdewz](https://osu.ppy.sh/users/10000899), ::{ flag=VN }:: [SIay](https://osu.ppy.sh/users/9587896), ::{ flag=US }:: [this1neguy](https://osu.ppy.sh/users/1797189) |
```

**Review by iSlodinxOsu — CHANGES_REQUESTED** [review]

File: wiki/Tournaments/4WC/2024/en.md (hunk 7 above)
Comment (shdewz) [reply wiki/Tournaments/4WC/2024/en.md:48]: good catch, still had 2023 data on my staff list it seems

**Review by peppy — APPROVED** [review]

**Pull request merged**